eiffel-vocabulary/EiffelServiceResumedEvent.md

```diff
@@ -0,0 +1,20 @@
+
+# EiffelSeriveResumedEvent (ArtS)
+
+The EiffelServiceResumedEvent declares that a service has been Resumed and is ready for use.
```

Comment:
thanks, I have made an update..
"The EiffelServiceResumedEvent declares that a service has been Resumed from a machine or human instructed Paused state (EiffelServicePausedEvent) and is now available once again for use."